src/PresentationalComponents/Inventory/Inventory.js

```diff
@@ -319,14 +320,16 @@ const Inventory = ({
         key: 'impacted_date',
         title: 'First Impacted',
         sortKey: 'impacted_date',
-        transforms: [sortable, wrappable],
-        props: { width: 15 },
+        transforms: [wrappable],
+        props: { width: 15, isStatic: true },
         renderFunc: lastSeenColumn.renderFunc,
       };
 
       systemProfile = {
         ...systemProfile[0],
+        sortKey: 'rhel_version',
```

Comment:
this is more of a placeholder for now, ATM I've made this unsortable as it is not supported by the backend. Eventually this sortKey might be system_porfile or something but its TBA.